bin/install-site.sh

```diff
@@ -307,6 +323,15 @@ Pin-Priority: 50
 EOF
     fi
 
+    if [ x"$DISTRIBUTION" = x"debian" ] && [ x"$DISTVERSION" = x"jessie" ]
```

Comment:
Have removed the `debian.mysociety.org` jessie-specific from the commit, looks like it was needed